vnext/Desktop/WinHTTPWebSocketResource.h

```diff
@@ -0,0 +1,100 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// clang-format off
+#pragma once
+
+#include <IWebSocketResource.h>
+
+// Windows API
+#include <winhttp.h>
+
+// Standard Library
+#include <functional>
+
+namespace Microsoft::React
+{
+class WinHTTPWebSocketResource : public IWebSocketResource
+{
+  static VOID CALLBACK Callback(HINTERNET handle, DWORD_PTR context, DWORD status, LPVOID info, DWORD infoLength);
```

Comment:
>VOID [](start = 9, length = 4)

prefer using native types when possible instead of typedefs

Reply:
Even if those are used in the MS docs and defined in the WinHTTP headers?